File: RaceLaneManager/Model/RaceLaneAssignment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RaceLaneManager.Model
{
    public class RaceLaneAssignment
    {
        public int RaceNum { get; set; }
        public Lane Lane { get; set; }
        public Racer Racer { get; set; }
        public long ElapsedTime { get; set; }
    }
}

File: website/models/race-lane-assignment.ts
class RaceLaneAssignment {
    raceNum: number;
    lane: Lane;
    car: Car;
    elapsedTime: number;

    static fromPayload(payload: any): RaceLaneAssignment {
        let assignment = new RaceLaneAssignment();

        assignment.raceNum = payload.RaceNum;
        assignment.lane = Lane.fromPayload(payload.Lane);
        assignment.car = Car.fromPayload(payload.Car);
        assignment.elapsedTime = payload.ElapsedTime;

        return assignment;
    }
}